core/modules/ccontrol/UserQuery.h

```diff
@@ -38,19 +38,26 @@
 
 // Qserv headers
 #include "ccontrol/QueryState.h"
+#include "ccontrol/UserQueryResources.h"
 #include "global/intTypes.h"
+#include "qmeta/types.h"
 
 // Forward decl
 namespace lsst {
 namespace qserv {
+namespace ccontrol {
+    class UserQueryResources;
```

Comment:
You do both forward-declare `UserQueryResources` and include its header, but it's not even used in this header.

Reply:
lulz